<?xml version="1.0"?>
<launch>

  <rosparam param="/vive_pose_converter/x_min">-1</rosparam>
  <rosparam param="/vive_pose_converter/x_max">1</rosparam>
  <rosparam param="/vive_pose_converter/y_min">-1</rosparam>
  <rosparam param="/vive_pose_converter/y_max">1</rosparam>
  <rosparam param="/vive_pose_converter/z_min">-1</rosparam>
  <rosparam param="/vive_pose_converter/z_max">1</rosparam>

  <node name="vive_pose_converter_node" pkg="sven_ros" type="vive_pose_converter_node" output="screen" required="true">
  	<remap from="/vive_pose_converter/pose" to="/equilibrium_pose"/>
  	<remap from="/vive/pose1" to="/vive/pose1"/>
  </node>

</launch>

